src/components/Modals/SpendAssetModal.js

```diff
@@ -64,16 +72,51 @@ function SpendAssetModal({ closeModal, assetData }) {
       {errors?.recipient?.message
         && <div className={styles.error}>{errors.recipient.message}</div>}
 
-      <div className={styles.title}>Amount LLD</div>
+      <div className={styles.title}>
+        Amount
+        {' '}
+        {assetData.metadata.name}
+        {' '}
+        (
+        {assetData.metadata.symbol}
+        )
+      </div>
       <TextInput
         register={register}
         name="amount"
-        placeholder={`Amount ${assetData.metadata.symbol}`}
+        placeholder={`Amount ${assetData.metadata.name} (${assetData.metadata.symbol})`}
         required
       />
       { errors?.amount?.message
         && <div className={styles.error}>{errors.amount.message}</div> }
 
+      <div className={styles.title}>Spend description</div>
+      <TextInput
+        register={register}
+        validate={(textDescription) => Validator.validateLen(3, textDescription)}
+        name="description"
+        placeholder="Spend description"
+        required
+      />
+      { errors?.description?.message
+        && <div className={styles.error}>{errors.description.message}</div> }
+
+      <div className={styles.title}>Senate voting duration in days</div>
```

Comment:
I don't understand, why are we allowing user to change this? This is always 4 days.

What could be adjusted is the time allocated for getting this through the Congress, as this could be as short as 1 day if congress coordinates.

The 4 days is a blockchain requirement - we need 4 days between the congress motion gets executed and the time scheduler should execute the call. User can't decide that.

It it just a confusion of Congress/Senate and this should be `Congress voting duration in days`? But in the nodeRpcCall it indeed is put in the place that refers to the time that Senate has for veto, not motionDuration.

Can you describe what was the intent here?